tests/stress.rs

```diff
@@ -4489,13 +4495,13 @@ unsafe fn create_context() -> dc_context_t {
             .unwrap()
     );
 
-    ctx
+    TestContext { ctx: ctx, dir: dir }
 }
 
 #[test]
 fn run_stress_tests() {
     unsafe {
-        let ctx = create_context();
-        stress_functions(&ctx);
+        let context = create_test_context();
+        stress_functions(&context.ctx);
```

Comment:
@dignifiedquire I couldn't make tuples work (probably did something wrong), but wrapping this in a struct worked well. At least now we can start splitting the tests. Is this good enough?

Reply:
looks good to me